packages/roosterjs-editor-plugins/lib/plugins/ImageEdit/imageEditors/Resizer.ts

```diff
@@ -142,18 +138,19 @@ export function getCornerResizeHTML(
 
     Xs.forEach(x =>
         Ys.forEach(y => {
-            let elementData = (x == '') == (y == '')
-            ? getResizeHandleHTML(
-                  x,
-                  y,
-                  resizeBorderColor,
-                  handlesExperimentalFeatures
-                      ? HandleTypes.CircularHandlesCorner
-                      : HandleTypes.SquareHandles
-              )
-            : null;
+            let elementData =
+                (x == '') == (y == '')
+                    ? getResizeHandleHTML(
+                          x,
+                          y,
+                          resizeBorderColor,
+                          handlesExperimentalFeatures
+                              ? HandleTypes.CircularHandlesCorner
+                              : HandleTypes.SquareHandles
+                      )
+                    : null;
             if (onShowResizeHandle) {
-                onShowResizeHandle(elementData, x, y)
+                onShowResizeHandle(elementData, x, y);
```

Comment:
elementData can be null, should we add a null check?

Reply:
I think this had to be node in another PR to enable strict mode. I didn't made any changes in this files, but the beautify extension add the semicolon